test/hts-precompile/utils.js

```diff
@@ -579,6 +579,10 @@ class Utils {
     }
   }
 
+  static keyListFromContractAddressAndKey(key, contract) {
```

Comment:
Was wondering why these changes are here in this PR? Is it related to the local-node being bumped to 2.25.2?